src/qtism/data/rules/BranchRuleCollection.php

```diff
@@ -31,6 +31,20 @@
  */
 class BranchRuleCollection extends QtiComponentCollection
 {
+
+    /** @var bool */
+    private $supportsNonLinearNavigationMode = false;
+
+    public function isAllowedForNonLinearNavigationMode(): bool
+    {
+        return $this->supportsNonLinearNavigationMode;
+    }
+
+    public function allowForNonLinearNavigationMode(): void
```

Comment:
Another suggestion.
```suggestion
    public function allowNonLinearNavigationMode(): void
```

"Allow for" sounds more like "take into consideration" in this conext.

Reply:
Done, thanks!